CoreFunctionality/test/unit/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettingsNGTest.java

```diff
@@ -42,93 +41,196 @@ public class ConstellationLAFSettingsNGTest {
     private static final Logger LOGGER = Logger.getLogger(ConstellationLAFSettingsNGTest.class.getName());
     final static PseudoLookAndFeel pseudoLAF = new PseudoLookAndFeel();
     final static PseudoUIDefaults pseudoUIdefaults = new PseudoUIDefaults();
-    
+    final static PseudoViewTabDisplayerUI pseudoViewDisplayerUI = new PseudoViewTabDisplayerUI();
+    final static PseudoTabDisplayerUI pseudoTabDisplayerUI = new PseudoTabDisplayerUI();
+    final static Class<?> pseudoViewDisplayerUIclass = pseudoViewDisplayerUI.getClass();
+    final static Class<?> pseudoTabDisplayerUIclass = pseudoTabDisplayerUI.getClass();
+
     @Test
-    public void runSetLAFTabColors() {
+    public void runSetWindowsXPTabColors() {
         try (final MockedStatic<WindowManager> windowManagerMockedStatic = Mockito.mockStatic(WindowManager.class)) {
             final WindowManager windowManager = mock(WindowManager.class);
             windowManagerMockedStatic.when(WindowManager::getDefault).thenReturn(windowManager);
-            
-            LOGGER.info("TESTING ConstellationLAFSettings ...");
-            UIManager.setLookAndFeel(pseudoLAF);
             final JFrame frame = mock(JFrame.class);
             when(windowManager.getMainWindow()).thenReturn(frame);
-
-            PseudoViewTabDisplayerUI pseudoViewDisplayerUI = new PseudoViewTabDisplayerUI();
-            PseudoTabDisplayerUI pseudoTabDisplayerUI = new PseudoTabDisplayerUI();
-            Class<?> pseudoViewDisplayerUIclass = pseudoViewDisplayerUI.getClass();
-            Class<?> pseudoTabDisplayerUIclass = pseudoTabDisplayerUI.getClass();
-
-            // Test XP LAF settings
+            
+            LOGGER.info("TESTING ConstellationLAFSettings for Windows XP ...");
+            UIManager.setLookAndFeel(pseudoLAF);
             pseudoLAF.setName("Windows XP");
             UIManager.getDefaults().put("org.netbeans.swing.tabcontrol.plaf.Windows8VectorViewTabDisplayerUI", null);
             UIManager.getDefaults().put("org.netbeans.swing.tabcontrol.plaf.WinXPViewTabDisplayerUI", pseudoTabDisplayerUIclass);
-
             ConstellationLAFSettings.applyTabColorSettings();
             final Color selectionIndicator = (Color) UIManager.getDefaults().get("TabbedPane.selectionIndicator");
             final Color expectedSelectionIndicator = new Color(180, 220, 255);                                                                
             Assert.assertTrue(selectionIndicator != null && selectionIndicator.equals(expectedSelectionIndicator));                                                              
-            LOGGER.info("Testing LAF: Windows XP : PASSED");
+            LOGGER.info("Windows XP LAF Test: PASSED");
 
-            // Test Win 8+ LAF settings
+        } catch (Exception e) {
+            LOGGER.log(Level.SEVERE, "\n******* EXCEPTION *******\n", e);            
+        }
```

Comment:
are you able to throw the exception instead? just logging the exception won't actually cause the test fail, which seems to be what you'd want to happen if you got here

Similar case for the other tests

Reply:
True, any exceptions caught should result in a failed test.
We need to maintain the catch block to handle any exceptions from the UIManager.setLookAndFeel() call, so an Assert.fail() has been added to the catch block.